test cases/common/122 llvm ir and assembly/meson.build

```diff
@@ -34,6 +34,9 @@ foreach lang : ['c', 'cpp']
       ml = find_program('ml', required: false)
     elif cpu == 'x86_64'
       ml = find_program('ml64', required: false)
+    elif cpu == 'aarch64'
+      # XXX: no idea what assembler we should use here
```

Comment:
armasm64 is what you want: https://gitlab.freedesktop.org/gstreamer/meson-ports/libffi/-/blob/meson/src/meson.build#L104-132

Reply:
Thanks.